.devcontainer/docker-compose.yml

```diff
@@ -14,8 +14,6 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-version: '3'
```

Comment:
Removed version from all our docker-compose files, because it's considered a long deprecated field and newer Docker installs are annoying about pointing that out.